app/models/metric/chargeback_helper.rb

```diff
@@ -21,11 +21,11 @@ def tag_prefix
   def tag_list_reconstruct
     tag_list = tag_names
 
-    if resource_type == Container.name
+    if resource.instance_of?(Container)
```

Comment:
I am not sure about it I thought that `resource` is preloaded in by `include` `base_rollup` query, isn't ?

https://github.com/ManageIQ/manageiq/blob/master/app/models/chargeback.rb#L26